vignettes/col_formatting.Rmd

```diff
@@ -55,6 +63,7 @@ library(rlistings)
 For this example, we will use the dummy ADAE dataset provided within the `formatters` package as our data frame, which consists of 48 columns of adverse event patient data, and one or more rows per patient. For the purpose of this example, we will subset the data and only use the first 15 records of the dataset. We will create some `NA` values in the data to showcase how `NA` values can be formatted, and sort the data by what will be our key columns.
 
 ```{r}
+require(dplyr)
```

Comment:
it seems that there is a dedicated library-calls chunk in lines 59-61 - let's add it there

Reply:
actually - do we use `dplyr` in that vignette at all? I cannot identify any dplyr call in here. As opposed to the `vignettes/ref_footnotes.Rmd` where we use it.

If it's not used then let's not introduce conditional eval for simplicity

Reply:
arrange

Reply:
mutate

Reply:
ok